examples/map-proof.js

```diff
@@ -3,7 +3,7 @@
  */
 'use strict'
 
-const Exonum = require('..')
+const Exonum = require('../')
```

Comment:
Not sure I understand the reason for this change.

Reply:
Test does not pass otherwise.

Reply:
That's quite strange, given that the PR that introduced them (#85) [has successfully passed CI](https://travis-ci.org/exonum/exonum-client/builds/364723304). If you are talking about the CI build (say, [this one](https://travis-ci.org/exonum/exonum-client/jobs/381439973)), the `node examples/map-proof.js` step doesn't pass because the code isn't transpiled on the previous step (due to linter errors), which results in missing `lib` dir, from which Node would load the package as instructed by `package.json`. As far as I can tell, using `..` or `../` has nothing to do with this outcome.

On the other hand, if you are unable to run tests locally, could you please specify your Node version? The example works for me locally for 4, 6, 8, and 10 (anecdotal evidence, I know).

Reply:
Here's the log from TravisCI: https://travis-ci.org/exonum/exonum-client/jobs/381439973

Reply:
The reason the job fails is like I said:

> the `node examples/map-proof.js` step doesn't pass because the code isn't transpiled on the previous step (due to linter errors), which results in missing lib dir, from which Node would load the package as instructed by `package.json`

I think the reason for confusion may be that the `npm install` step in CI does not perform transpilation in Node 4 and 6 jobs (i.e., with `npm < 4.0`), because transpilation is specified in `package.json` in [the `prepare` script, rather than `prepublish` one](https://docs.npmjs.com/misc/scripts#prepublish-and-prepare). Notice that the same Travis build has *errorred* (rather than *failed*) for Node 8, because it *does* invoke the `prepare` script during `npm install` (which fails). So, Node 4 and 6 CI jobs end up without the transpiled package while executing `node examples/map-proof.js`, because the `grunt` command that is supposed to build the package has failed. To reiterate, it has nothing to do with `require(...)` statements, and everything to do with the fact that the source code for the package had lints.